lib/miq/md_link_converter.rb

```diff
@@ -24,10 +26,21 @@ def with_prefixed_path
 
     private
 
+    def external_link?(url)
+      begin
+        uri = URI(url)
+      rescue
+        return false # in case caller does not pass in valid URL
+      end
+      return false if uri.scheme.nil? || uri.host.nil?
```

Comment:
I think there's a `uri.relative?` that might be cleaner than checking manually.

Reply:
`uri.relative?` is the same as `uri.scheme.nil?`.  Would still need to check `uri.host.nil?`, I believe